src/modules/75-cd/components/PipelineSteps/Common/ProvisionerConstants.ts

```diff
@@ -7,7 +7,26 @@
 
 export enum ProvisionerTypes {
   Terraform = 'Terraform',
+  Terragrunt = 'Terragrunt',
   CloudFormation = 'CloudFormation',
   ARM = 'ARM',
+  BLUEPRINT = 'Blueprint',
```

Comment:
Nit: this can also be in the same case as the rest of the entries
```suggestion
  Blueprint = 'Blueprint',
```